torch/_inductor/async_compile.py

```diff
@@ -219,6 +219,7 @@ def triton(self, kernel_name: str, source_code: str, device_str: str = "cuda"):
 
         kernel = TritonCodeCache.load(kernel_name, source_code)
         if self._use_process_pool():
+            set_feature_use("pytorch/inductor:enable_parallel_compile_version", True)
```

Comment:
This one is a little weird because _use_process_pool might return False when we _Want_ to use the pool, but it's not warmed up yet. Maybe just do this above the if block:
```
set_feature_use("pytorch/inductor:enable_parallel_compile_version", get_compile_threads() > 1)
```

Reply:
Talked offline, add a "(post warmup" to the name.

Reply:
Done.